assets/js/modules/adsense/components/settings/SettingsView.js

```diff
@@ -61,6 +61,9 @@ export default function SettingsView() {
 	const accountStatus = useSelect( ( select ) =>
 		select( MODULES_ADSENSE ).getAccountStatus()
 	);
+	const adBlockerDetectionSnippet = useSelect( ( select ) =>
```

Comment:
This should ideally be consistent with the setting it represents.
```suggestion
	const useAdBlockerDetectionSnippet = useSelect( ( select ) =>
```